static/templates/admin_user_list.handlebars

```diff
@@ -43,15 +43,27 @@
     </div>
   </td>
 </tr>
-{{#if is_active}}
-<tr class="user-name-form display-none" id="user_form_{{email}}">
-  <td colspan="{{#if is_bot}}4{{else}}3{{/if}}">
+{{/with}}
+{{#if user.is_active}}
+<tr class="user-name-form display-none" id="user_form_{{user.email}}">
+  <td colspan="{{#if user.is_bot}}4{{else}}3{{/if}}">
     <form class="form-horizontal name-setting">
       <input type="hidden" name="is_full_name" value="true" />
       <div class="input-group name_change_container">
         <label for="full_name">{{t "Full name" }}</label>
-        <input type="text" name="full_name" value="{{ full_name }}" />
+        <input type="text" name="full_name" value="{{ user.full_name }}" />
       </div>
+      {{#if user.is_bot}}
+        <div class="input-group">
+          <label for="owner">{{t "Owner" }}</label>
+          <select name="owner" id="edit_bot_owner_{{user.email}}">
+            <option value=''></option>
+            {{#each users_list}}
+              <option value='{{this.email}}'>{{this.full_name}}</option>
```

Comment:
Can we use user.user_id instead of user.email in HTML attributes such `id=edit_bot_owner_` and `option value=`?

Also have you tested this on a large realm with maybe 500 users to see how the UI feels with a big dropdown here?  You can use `./manage.py populate_db --extra-users 500` to create a realm with lots of users.